addons/payment_razorpay/static/src/js/razorpay.js

```diff
@@ -0,0 +1,83 @@
+odoo.define('payment_razorpay.razorpay', function(require) {
+    "use strict";
+
+    var ajax = require('web.ajax');
+    var core = require('web.core');
+    var _t = core._t;
+    var qweb = core.qweb;
+    ajax.loadXML('/payment_razorpay/static/src/xml/payment_razorpay_templates.xml', qweb);
+
+    require('web.dom_ready');
+    if (!$('.o_payment_form').length) {
+        return $.Deferred().reject("DOM doesn't contain '.o_payment_form'");
+    }
+
+    var observer = new MutationObserver(function(mutations, observer) {
+        for(var i=0; i<mutations.length; ++i) {
+            for(var j=0; j<mutations[i].addedNodes.length; ++j) {
+                if(mutations[i].addedNodes[j].tagName.toLowerCase() === "form" && mutations[i].addedNodes[j].getAttribute('provider') == 'razorpay') {
+                    display_razorpay_form($(mutations[i].addedNodes[j]));
+                }
+            }
+        }
+    });
+
+    function razorpay_show_error(msg) {
+        var wizard = $(qweb.render('razorpay.error', {'msg': msg || _t('Payment error')}));
+        wizard.appendTo($('body')).modal({'keyboard': true});
+    };
+
+    function razorpay_handler(resp) {
+        if (resp.razorpay_payment_id) {
+            $.post('/payment/razorpay/capture',{
+                payment_id: resp.razorpay_payment_id,
+            }).done(function (data) {
+                window.location.href = data;
+            }).fail(function (data) {
+                razorpay_show_error(data && data.data && data.data.message);
+            });
+        }
+    };
+
+    function display_razorpay_form(provider_form) {
+        // Open Checkout with further options
+        var payment_form = $('.o_payment_form');
+        if(!payment_form.find('i').length)
```

Comment:
Is there some missing brackets here? `payment_form.attr('disabled','disabled');` is indented as inside this `if`